pkg/proto/block.go

```diff
@@ -423,6 +422,25 @@ type Block struct {
 	Transactions Transactions `json:"transactions,omitempty"`
 }
 
+func (b *Block) MarshalJSON() ([]byte, error) {
```

Comment:
Do we have test on block marshalling on API level? I think we have to check that there is no unnecessary levels in block's json.

Reply:
Yep, we have. See `blockFromJSONToJSON` function in `pkg/proto/block_test.go`.

Reply:
Fixed. Tests have been added.